assets/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(TangoSourceDistribution)
 
+set(IDL_BASE ${PROJECT_SOURCE_DIR}/lib/idl)
```

Comment:
Why not install the IDL somewhere and use this installation directory as IDL_BASE?
Then we don't need to change IDL_BASE definition in cppTango...

Reply:
Sure, I am just playing with it. Basically we need to install not only idl, but tango, before we can try to  compile other Tango components